src/streamlink/plugins/douyutv.py

```diff
@@ -109,9 +108,11 @@ def _get_streams(self):
         res = http.get(MAPI_URL.format(channel))
         room = http.json(res, schema=_room_schema)
         if not room:
+            self.logger.info("API error")
```

Comment:
Can we be more descriptive with this error? `API error` isn't very helpful for users.